mantra.md

```diff
@@ -22,10 +22,43 @@
 8. ACK Triangle
 ===============
 
+Contrary to belief of many new-age engineering shops; the utmost elegant
+products must be built in a environment constricted by bureaucratic hurdles.
```

Comment:
* Semicolon should be a comma since the part on its left is not a standalone sentence.
* I'd say "products of utmost elegance must be built..."